test/metabase/query_analysis/native_query_analyzer_test.clj

```diff
@@ -0,0 +1,54 @@
+(ns metabase.query-analysis.native-query-analyzer-test
```

Comment:
This just moved, but since it also had some tests moved out of it git is treating it as an add-and-remove.